scss/_popover.scss

```
@@ -203,15 +209,19 @@
   &:empty {
     display: none;
   }
-
-  // Boosted mod: popover body special padding when following a popover header
-  + .popover-body {
-    padding-top: var(--#{$prefix}popover-header-padding-bottom);
-  }
-  // End mod
 }
 
 .popover-body {
-  // Boosted mod: no `padding`
+  padding:
+    var(--#{$prefix}popover-body-padding-y, var(--#{$prefix}popover-body-padding-top))
+    var(--#{$prefix}popover-body-padding-x)
+    var(--#{$prefix}popover-body-padding-y, var(--#{$prefix}popover-body-padding-bottom))
+    var(--#{$prefix}popover-body-padding-x); // Boosted mod: instead of padding: `var(--#{$prefix}popover-body-padding-y) var(--#{$prefix}popover-body-padding-x);`
```

Maybe it should be on one line ? Or don't we care ?

Should we explain this behavior in the doc/migration ?

Anyway proposal to remove second call to `var(--#{$prefix}popover-body-padding-x)`. Same above ?

```suggestion
  padding:
    var(--#{$prefix}popover-body-padding-y, var(--#{$prefix}popover-body-padding-top))
    var(--#{$prefix}popover-body-padding-x)
    var(--#{$prefix}popover-body-padding-y, var(--#{$prefix}popover-body-padding-bottom)); // Boosted mod: instead of padding: `var(--#{$prefix}popover-body-padding-y) var(--#{$prefix}popover-body-padding-x);`
```

---

Yep let's explain it in the "CSS variables" section after the list of custom props rather than in the migration guide.
And yeah, let's remove the latest useless `var(--#{$prefix}popover-body-padding-x)`.